src/fetch/.gitignore

```diff
@@ -0,0 +1,2 @@
+__pycache__
+.venv
```

Comment:
this should probably a standard python project one as the ones github would create.